worktree_status.go

```diff
@@ -371,7 +371,7 @@ func (w *Worktree) doAdd(path string, ignorePattern []gitignore.Pattern, skipSta
 	}
 
 	if err != nil || !fi.IsDir() {
-		added, h, err = w.doAddFile(idx, s, path, ignorePattern)
+		added, h, err = w.doAddFile(idx, s, filepath.Clean(path), ignorePattern)
```

Comment:
Please update this to `path = filepath.Clean(path)` above the `if`, as per change against `v6-exp`.

Reply:
OK done.